src/main/app/logging/httpProxyCallInterceptor.ts

```diff
@@ -0,0 +1,17 @@
+export const HttpMethods = ['get', 'post', 'put', 'patch', 'delete', 'del', 'head']
+
+export type CallHandler = (target: Object, key: string, callArgs: any[]) => void
+
+export class HttpProxyCallInterceptor {
+  static intercept (target: Object, key: string, handler: CallHandler): any {
```

Comment:
it returns a function or something else?
can we do better than `any`?

Reply:
It's a handler for [`Proxy.get`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Proxy/handler/get), could be anything.